package.json

```diff
@@ -53,7 +54,8 @@
         "zip",
         "AppImage"
       ],
-      "maintainer": "Akash Nimare <svnitakash@gmail.com>"
+      "maintainer": "Akash Nimare <svnitakash@gmail.com>",
+      "asarUnpack": "./app/node_modules/node-notifier/vendor/**"
```

Comment:
Do we really need to unpack this on Linux and Windows?

Reply:
Yep, The node-notifier says we need to unpack them.